src/main/java/com/yahoo/sketches/counting/FrequentItems.java

```diff
@@ -0,0 +1,107 @@
+package com.yahoo.sketches.counting;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+/**
+ * This class implements a simple frequent direction algorithm.
+ * It is not intended to be very fast but rather to provide a 
+ * correctness baseline for faster versions 
+ * 
+ * The algorithm is most commonly known as the "Misra-Gries algorithm", 
+ * "frequent items" or "space-saving". It was discovered and rediscovered and redesigned several times over the years.
+ * "Finding repeated elements", Misra, Gries, 1982 
+ * "Frequency estimation of internet packet streams with limited space" Demaine, Lopez-Ortiz, Munro, 2002
+ * "A simple algorithm for finding frequent elements in streams and bags" Karp, Shenker, Papadimitriou, 2003
+ * "Efficient Computation of Frequent and Top-k Elements in Data Streams" Metwally, Agrawal, Abbadi, 2006
+ * 
```

Comment:
Your "What it does" section from your Readme.md also belongs here.  This would eliminate the readme file, which really doesn't belong in the source code tree.